db/seeds/dev/meal_foods.js

```diff
@@ -0,0 +1,47 @@
+exports.seed = (knex, Promise) => {
+  return knex.raw('TRUNCATE meal_foods RESTART IDENTITY')
```

Comment:
In reference to the previous comment, when you had REFERENCES instead of INDEX, I suggested that we might clear the meal_foods table before clearing meals and foods table - in that order. That would require incorporating all the seed files in the development directory into one single file. I wonder if having everything broken out into separate seed files for each object/table the way we have now is general practice or if its actually the opposite - having one main seed file.

Reply:
Yes! I think that's a great idea to get all the prep for the DB into a single file